test/functional/tests/lazy_writes/test_lazy_writes_clean.py

```diff
@@ -0,0 +1,369 @@
+            TestRun.LOGGER.error("Md5 sums should be equal.")
+
+    with TestRun.step("Read data from the core device."):
+        test_file_2 = File.create_file("/tmp/test_file_2")
+        dd = Dd().output(test_file_2.full_path) \
+            .input(core_part.system_path) \
+            .block_size(bs) \
+            .count(int(test_file_main.size / bs)) \
+            .oflag("direct")
+        dd.run()
+        test_file_2.refresh_item()
+        sync()
+
+    with TestRun.step("Compare md5 sum of test files."):
+        if not is_equal(test_file_md5sum_main, test_file_2.md5sum()):
+            TestRun.LOGGER.error("Md5 sums should be equal.")
+
+    with TestRun.step("Check IO stats generated for the core device since last check."):
+        core_disk_writes_current = (
+                check_device_write_stats(core_part) - core_disk_writes_initial
+        )
+
+        if is_zero(core_disk_writes_current):
+            TestRun.LOGGER.error("Writes should occur on the core device after stopping cache.")
+        if not is_equal(core_disk_writes_current, exp_obj_writes_current):
+            TestRun.LOGGER.error("Write statistics for the core device should be equal "
+                                 "to those from the exported object.")
+
+    with TestRun.step("Delete test files."):
+        test_file_main.remove(True)
+        test_file_1.remove(True)
+        test_file_2.remove(True)
+
+
+@pytest.mark.parametrize("fs", Filesystem)
+@pytest.mark.parametrize("cache_mode", CacheMode.with_traits(CacheModeTrait.LazyWrites))
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_clean_remove_core_with_fs(cache_mode, fs):
+    """
+        title: Test of the ability to remove core from cache in lazy-write modes with filesystem.
+        description: |
+          Test if OpenCAS removes core from cache in modes with lazy writes and with different
+          filesystems without data loss.
+        pass_criteria:
+          - Core removing works properly.
+          - Data on core device is correct after core is removed.
+    """
+    with TestRun.step("Prepare devices for cache and core."):
+        cache_dev = TestRun.disks['cache']
+        cache_dev.create_partitions([Size(256, Unit.MebiByte)])
+        cache_part = cache_dev.partitions[0]
+        core_dev = TestRun.disks['core']
+        core_dev.create_partitions([Size(512, Unit.MebiByte)])
+        core_part = core_dev.partitions[0]
+        Udev.disable()
+
+    with TestRun.step(f"Start cache in {cache_mode} mode."):
+        cache = casadm.start_cache(cache_part, cache_mode)
+
+    with TestRun.step(f"Add core with {fs.name} filesystem to cache and mount it."):
+        core_part.create_filesystem(fs)
+        core = cache.add_core(core_part)
+        core.mount(mnt_point)
+
+    with TestRun.step("Disable cleaning and sequential cutoff policies."):
```

Comment:
You cannot disable policies - you can change policies, thus disabling cleaning and sequential cutoff. So the word "policies" should be removed.